### ISSUE TITLE
Droidian Image

### ISSUE THREAD

**Issue by anonimo82** [Exynos7880-Linux/samsung-exynos7880#2]
Please, I tried several times no stop from this morning around 09:00 until now (19:00), and I had no luck at all.
Can you put together a Droidian image for A5/A7 TWRP?

Thanks a lot if you do.

**Comment by aymanrgab** [comment]
Did you follow the steps?

**Comment by anonimo82** [comment]
Here's what I did:
- Copy vendor.img to /data
- Open a terminal from TWRP
- in such terminal, `mkdir -p /vendor && mount /data/vendor.img /vendor/`
- Flash the GSI image
- Flash your boot.img
- Reboot

Vendor.img I got from Telegram

**Comment by aymanrgab** [comment]
It didn't booted? Stuck on Plymouth screen or what ?

**Comment by anonimo82** [comment]
Stuck at a screen with the Droidian logo (similar to the Debian one but green)
And no activity at all (neither blinking logo)

Maybe I got the wrong files?

**Comment by aymanrgab** [comment]
I guess it didn't generate a udev rules file you can download it from this repo overlay/system/usr/lib/udev/rules.d/70-exynos7880.rules and copy it in droidian rootfs file in /etc/udev/rules.d/

**Comment by anonimo82** [comment]
I don't understand. Should I do from the file manager in TWRP?

**Comment by aymanrgab** [comment]
`mkdir droidian && mount /data/rootfs.img droidian`
And copy the file to /droidian/etc/udev/rules.d/

**Comment by anonimo82** [comment]
Where should I get that rootfs.img?

**Comment by aymanrgab** [comment]
It's sometimes stuck on Plymouth cuz it can't mount android rootfs run `e2fsck -fy /droidian/var/lib/lxc/android/android-rootfs.img`

**Comment by aymanrgab** [comment]
> Where should I get that rootfs.img?

From droidian zip file!!

**Comment by anonimo82** [comment]
Oh, ok
@aymanrgab Please, can you summarize the steps?

**Comment by anonimo82** [comment]
And put them in the right order

**Comment by anonimo82** [comment]
> It's sometimes stuck on Plymouth cuz it can't mount android rootfs run `e2fsck -fy /droidian/var/lib/lxc/android/android-rootfs.img`

I don't have that folder in my system, the tree stops at /var/lib

**Comment by anonimo82** [comment]
Still no luck, same result as before. I copied the file you told, I did e2fsck, but I still have that issue.
